### ISSUE TITLE
Losslessly optimize images

### ISSUE THREAD

**Issue by coliff** [swapmyvote/swapmyvote#202]
Reducing file-size images with no loss in quality will improve load times = happy users.

You could do them manually with [imageoptim](https://imageoptim.com/mac) If you've got a Mac or with the https://squoosh.app/ web-app.

Overall I recommend the free GitHub [Imgbot app](https://imgbot.net/). It works automatically and will open a PR for you.

**Comment by aspiers** [comment]
Sounds good.  Imgbot is $10/month but I assume we would only need it for a day or two while we squash images?  Again adding labels and/or submitting PRs would be hugely appreciated here :-)

**Comment by coliff** [comment]
imgbot is free for public and open source projects. It's a nobrainer :-)  I'm not an admin on this project so can't do it myself - you just install the app and it'll do it all itself.

**Comment by aspiers** [comment]
No, it's free for open source projects but we're not open source.

**Comment by fkorning** [comment]
imagemagick.  not sur ewhat you mean by losslessly optimize here.  you mean up the compression ratios (not lossless), or do you mean make them load as progressive images ("faster", smoother rendering).

**Comment by coliff** [comment]
There are many open-source libraries which losslessly optimize images.. For example OptiPNG (http://optipng.sourceforge.net/). Using that on https://www.swapmyvote.uk/assets/logo_nav-4ea720648e125fcceb438500b7a72067c24f1a36c95d76324527854fa7340055.png makes the file 21% smaller. It's lossless - no image information is lost at all. You can test it at: https://squoosh.app/

**Comment by fkorning** [comment]
that's me educated. thanks.
